scheduler/planner.go

```diff
@@ -228,7 +228,7 @@ func (unit *Unit) RankValue() int64 {
 		priority += length
 	}
 
-	if inPatch {
+	if inPatch || inCommitQueue {
```

Comment:
This LGTM, but could you add a comment? I could see someone removing this because it's not obvious why it's there.

Reply:
added comment below